cc/tf_bt_utils.cc

```diff
@@ -127,6 +128,46 @@ void WriteGameExamples(const std::string& gcp_project_name,
             << examples.size() << std::endl;
 }
 
+void WriteEvalRecord(const std::string& gcp_project_name,
+                     const std::string& instance_name,
+                     const std::string& table_name,
+                     const MctsPlayer& player,
+                     const std::string& black_player_name,
+                     const std::string& white_player_name) {
+  // TODO assert("eval" in table_name);
```

Comment:
I want to make sure we don't accidentally write to the wrong table. not sure best way to prevent that.

Reply:
If I understand the rest of the code, it looks like you're only wanting to add one new evaluation row per game, with the prefix `e_`. If so, it can simply be another keyspace within the same table, like the `ct_` keyspace for move counts.